.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -23,7 +24,7 @@ For OEM releases, keep the OEM Release template and remove the Open Release one
  - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder
  - [ ] [DIS] Move all changelog files from the `unreleased` folder to the new version folder
  - [ ] [DIS] Update screenshots, if needed, in README.md
```

Comment:
So this step is not needed anymore, right? We can merge it with `[DIS] Update screenshots and check they are OK in Play Store`. I would modify that step like I indicate in the next comment

Reply:
that was a leftover from other modifications. Yes, that step should be out from there.